features/uri-templating.feature

```diff
@@ -2,68 +2,79 @@
+    En een Adres is te bevragen bij de BAG API via endpoint '/adressen'
+    Als een templated Hal link voor de Adres is gegenereerd
+    Dan is de Hal link naar de Adres gelijk aan
+    | href                                         | templated |
+    | {bagserverurl}/adressen/{adresIdentificatie} | true      |
+  
+  Scenario: Verwijzing naar één externe Resource via gegevensgroep property
+    Gegeven een KadastraalOnroerendeZaak heeft een verwijzing via de 'adresIdentificatie' property van gegevensgroep 'woonadres' naar een Adres
+    En een Adres is te bevragen bij de BAG API via endpoint '/adressen'
+    Als een templated Hal link voor de Adres is gegenereerd
+    Dan is de Hal link naar de Adres gelijk aan
+    | href                                                   | templated |
+    | {bagserverurl}/adressen/{woonadres.adresIdentificatie} | true      |
+
+  Scenario: Verwijzingen naar meerdere externe Resources van dezelfde soort
+    Gegeven een KadastraalOnroerendeZaak heeft een verwijzing via de 'adresIdentificaties' property naar meerdere Adressen
+    En een Adres is te bevragen bij de BAG API via endpoint '/adressen'
+    Als een templated Hal link voor de Adressen is gegenereerd
+    Dan is de Hal link naar de Adressen gelijk aan
+    | href                                          | templated |
+    | {bagserverurl}/adressen/{adresidentificaties} | true      |
 
-    Voorbeelden:
-    | type     | templated url                                                                                    |
-    | absoluut | https://{hostname}/lvbag/api/haal-centraal-bag-bevragen/v{version}/adressen/{adresidentificatie} |
-    | relatief | /lvbag/api/haal-centraal-bag-bevragen/v{version}/adressen/{adresidentificatie}                   |
+  Scenario: Verwijzing naar verschillende externe Resources
+    Gegeven een ZakelijkGerechtigde heeft een verwijzing via de 'identificatie' property naar een Ingeschreven Natuurlijk Persoon
+    En een Ingeschreven Natuurlijk Persoon is te bevragen bij de BRP API via de endpoint /ingeschrevenpersonen
+    En een ZakelijkGerechtigde heeft een verwijzing via de 'identificatie' property naar een Ingeschreven Niet Natuurlijk Persoon
+    En een Ingeschreven Niet Natuurlijk Persoon is te bevragen bij de BRK via de endpoint /kadasternietnatuurlijkpersonen
+    Als een templated Hal link voor de Personen is gegenereerd
+    Dan is de Hal link naar de Persoon van het type ingeschreven_natuurlijk_persoon gelijk aan
+    | href                                                | templated |
+    | {brpserverurl}/ingeschrevenpersonen/{identificatie} | true      |
+    En is de Hal link naar de Persoon van het type ingeschreven_niet_natuurlijk_persoon gelijk aan
+    | href                                                          | templated |
+    | {brkserverurl}/kadasternietnatuurlijkpersonen/{identificatie} | true      |
```

Comment:
een interne link zouden we toch als relatieve link opnemen?

Reply:
In mijn beeld zouden we idd een interne link als relatieve link opnemen. Hebben we dan uberhaupt nog de placeholder {brkserverurl nodig}?

Reply:
aangepast